scopesim/effects/effects.py

```diff
@@ -41,7 +47,11 @@ class Effect(DataContainer):
     required_keys = set()
 
     def __init__(self, filename=None, **kwargs):
-        super().__init__(filename=filename, **kwargs)
+        self.data_container = DataContainer(filename=filename, **kwargs)
+        self.meta = kwargs.get("meta", {})
+        self.cmds = kwargs.get("cmds")
+
+        self.meta.update(self.data_container.meta)
         self.meta["z_order"] = []
         self.meta["include"] = True
         self.meta.update(kwargs)
```

Comment:
It seems that this last `self.meta.update(kwargs)` is (currently) superfluous due to the `self.meta.update(self.data_container.meta)`. Nevertheless, I prefer to keep it as-is, because I would prefer that we, in due time, remove the `**kwargs` from `self.data_container = DataContainer(filename=filename, **kwargs)`.